doc/man3/SSL_write.pod

```diff
@@ -17,6 +18,14 @@ SSL_write_ex() and SSL_write() write B<num> bytes from the buffer B<buf> into
 the specified B<ssl> connection. On success SSL_write_ex() will store the number
 of bytes written in B<*written>.
 
+SSL_sendfile() writes B<size> bytes from offset B<offset> in the file
+descriptor B<fd> to the specified SSL connection B<s>. This function provides
+efficient zero-copy semantics. SSL_sendfile() is available only when
+Kernel TLS is enabled, which can be checked by calling BIO_get_ktls_send().
+It is provided here to allow users to maintain the same interface.
```

Comment:
I'm not sure I follow the meaning of this sentence. What is provided that allows the same interface to be maintained?

Reply:
The user could have used the sendfile system call directly on the file descriptor. But, this would prevent the handling of control messages.

SSL_sendfile allows application to handle control messages while using the sendfile interface.

Reply:
Fixed